server/exports.lua

```diff
@@ -145,9 +145,7 @@ exports('AddGangs', AddGangs)
 
 local function GetCoreVersion(InvokingResource)
     local resourceVersion = GetResourceMetadata(GetCurrentResourceName(), 'version')
-    if InvokingResource ~= nil then
-        print( ("%s called qbcore version check: %s"):format(InvokingResource, resourceVersion) )
-    end
+    print(("%s called qbcore version check: %s"):format(InvokingResource or 'Unknown Resource', resourceVersion))
```

Comment:
Left the if statement in specifically to nil check so they dont have to pass in the invoking resource, should've added a comment my bad.

Reply:
My bad, I overlooked this, will change this one back